### ISSUE TITLE
unsupported protocol scheme

### ISSUE THREAD

**Issue by DaPeace** [mcuadros/OctoPrint-TFT#14]
Ive copied plain octopi-image to sd-card, updated to the latest release and then run the steps from this repo to install Octoprint-tft.
When i now start up i see the octo-image and "connection to octoprint" and after some seconds i get following error in syslog and nothing happens:

time="2018-02-26T09:01:43Z" level=debug msg="Unexpected error: Get /api/connection: unsupported protocol scheme \"\"" file=ui.go func="ui.(*UI).verifyConnection" line=100

What im doing wrong?
Regards, Jan

**Comment by DaPeace** [comment]
I retried again with a vanilla octopi and have the same issue. No way to get it running without that message.. what am i doing wrong?

**Comment by CJeffyB** [comment]
Are you able to start the web interface of OctoPrint?
If so, you may try copying the API key from there, and manually entering it into the configuration file for OctoPrint-TFT...

**Comment by DaPeace** [comment]
Yes, the webinterface is working. but at the startscreen of octoprint-tft and in syslog i get "unsupported protocol scheme"
Ive tried to manually add the API-Key to the config-file and its the same error.

**Comment by CJeffyB** [comment]
Does the system keep retrying the connection, or does it fail once and that's it?

Have you tried another fresh install?  Or perhaps even compiling this plugin from source?  (I did that one, and it isn't terribly difficult.  Just takes a bit of time.)

Unfortunately, anything further is outside of my abilities to troubleshoot.  Hopefully @mcuadros has some idea or suggestion...

**Comment by mcuadros** [comment]
@DaPeace your `OCTOPRINT_HOST` it's prefixed with `http://`?

**Comment by DaPeace** [comment]
I will setup a new plain octoprint now and try out with http:// if that would help. I think i have only written localhost but im not sure.
The error message is flooding the logs so i guess its constantly trying to connect to octoprint.

**Comment by CJeffyB** [comment]
@DaPeace I'm guessing that the error code flooding the logs is related to [https://github.com/mcuadros/OctoPrint-TFT/issues/15](https://github.com/mcuadros/OctoPrint-TFT/issues/15), recently flagged as a definite bug...

**Comment by DaPeace** [comment]
@CJeffyB adding "http://localhost" instead of only "localhost" did the trick. I did not have a look at the tft yet but i reinstalled the raspberry remotely and now there is no error anymore.
